geo/src/algorithm/intersects.rs

```diff
@@ -211,32 +211,27 @@ where
     T: Float,
 {
     fn intersects(&self, bounding_rect: &Rect<T>) -> bool {
-        // line intersects inner or outer polygon edge
-        if bounding_rect.contains(self) {
-            false
```

Comment:
This contains check is backwards. If `a` contains `b` then we should always consider the rects to intersect.

We could fix it by changing this to return true, but it's wholly redundant with the subsequent check for overlap, so I removed it altogether.